.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: docs-deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+
+      - name: deps
+        run: make dev
+
+      - name: docs
+        run: make doc
+
+      - name: deploy
+        uses: peaceiris/actions-gh-pages@v3.8.0
```

Comment:
Instead of using this action, can we just do the push ourselves instead? E.g. something like https://stackoverflow.com/questions/57921401/push-to-origin-from-github-action/58393457#58393457

Reply:
Yeah, I'll give that a spin 🙂

Reply:
Done; let me know what you think of the approach. It's a little bit more involved than the SO snippets, since we need to create a blank-slate "orphan" branch to avoid pushing up lots of incidental files.